tests/integration_tests/packet_script.py

```diff
@@ -0,0 +1,80 @@
+import requests, argparse, sys, packet, time, uuid, os
```

Comment:
Don't you already have jumpscale installed for your test ? If, so this file is just re-inventing the wheel.
Also, I don't really see what packet is required at all to run the tests....

Reply:
On Travis i don't have jumpscale .. so I'm using this script .. Concerning why I'm using packet machine to lunch the test from there instead of using travis itself (which doesn't make sense )is because i can't get a js-ovc-client on the travis machine when it is dealing with zerotier .. @grimpy guessed it may be a problem with the packet size.